### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by iuliadmtru**
- The references to the old design were removed. All examples now use the new general design.
- The Basic usage section is now more complete.
- I made some rephrasing in the other sections.
- The Documentation section was removed. It is now referenced at the end of the Basic usage section.

### CODE REVIEW

**Comment by iuliadmtru** [comment]
This partially solves #173.

File: README.md [hunk 1]
```diff
@@ -6,111 +6,260 @@
 [![codecov](https://codecov.io/gh/xKDR/Survey.jl/branch/main/graph/badge.svg?token=4PFSF47BT2)](https://codecov.io/gh/xKDR/Survey.jl)
 [![Milestones](https://img.shields.io/badge/-milestones-brightgreen)](https://github.com/xKDR/Survey.jl/milestones)
 
+This package is used to study complex survey data. It aims to be a fast alternative
+to the [Survey package in R](https://cran.r-project.org/web/packages/survey/index.html)
+developed by [Professor Thomas Lumley](https://www.stat.auckland.ac.nz/people/tlum005).
 
-This package is used to study complex survey data. It aims to be a fast alternative to the [Survey package in R](https://cran.r-project.org/web/packages/survey/index.html) developed by [Professor Thomas Lumley](https://www.stat.auckland.ac.nz/people/tlum005).
+This package currently supports simple random sample, stratified sample, one- and
```
Comment (ayushpatnaikgit) [inline README.md:13]: It supports multistage sampling, but using single stage approximation.

Comment (iuliadmtru) [reply README.md:13]: Right. I will add a short sentence mentioning this with a reference to the documentation for more details. Right now we don't have more details in the documentation, but we will add it. So the reference will lead nowhere useful for now, but will definitely lead somewhere soon.

**Review by smishr — APPROVED** [review]
yes looks good.

File: README.md [hunk 2]
```diff
@@ -6,111 +6,263 @@
 [![codecov](https://codecov.io/gh/xKDR/Survey.jl/branch/main/graph/badge.svg?token=4PFSF47BT2)](https://codecov.io/gh/xKDR/Survey.jl)
 [![Milestones](https://img.shields.io/badge/-milestones-brightgreen)](https://github.com/xKDR/Survey.jl/milestones)
 
+This package is used to study complex survey data. It aims to be a fast alternative
+to the [Survey package in R](https://cran.r-project.org/web/packages/survey/index.html)
+developed by [Professor Thomas Lumley](https://www.stat.auckland.ac.nz/people/tlum005).
 
-This package is used to study complex survey data. It aims to be a fast alternative to the [Survey package in R](https://cran.r-project.org/web/packages/survey/index.html) developed by [Professor Thomas Lumley](https://www.stat.auckland.ac.nz/people/tlum005).
+This package currently supports simple random sample, stratified sample, single and
```
Comment (ayushpatnaikgit) [inline README.md:13]: It supports any kind of sample.

File: README.md [hunk 3]
```diff
@@ -6,111 +6,263 @@
...
+ Row │ total      SE
+     │ Float64    Float64
+─────┼──────────────────────
+   1 │ 3.62107e6  1.72846e5
+```
 
-mean(:api00, :cname, dsrs)
+Now we know the mean academic performance index from the year 2000 and the total
+number of students enrolled in the sampled Californian schools. We can also
+calculate the statistic of two variables in one go...
```
Comment (ayushpatnaikgit) [inline README.md:89]:
```suggestion
calculate the statistic of multiple variables in one go...
```

**Comment by ayushpatnaikgit** [comment]
It seems redundant to show mean, domain estimation etc for all the design. Perhaps show the constructor for srs, stratified sample, and cluster sample, but estimation for only 1.

File: README.md (hunk 2 above)
Comment (iuliadmtru) [reply README.md:13]: Would a stratified + cluster sample give correct results?

**Comment by iuliadmtru** [comment]
> It seems redundant to show mean, domain estimation etc for all the design. Perhaps show the constructor for srs, stratified sample, and cluster sample, but estimation for only 1.

Ok, I agree.

File: README.md (hunk 2 above)
Comment (iuliadmtru) [reply README.md]: @ayushpatnaikgit What do you think now?

**Comment by iuliadmtru** [comment]
I suggest merging #175 before merging this. The README.md file was not changed in that PR so there will be no conflicts. I can pull the changes from there to here and update the README according to the new `show` method.

**Comment by ayushpatnaikgit** [comment]
This will need to change according to PR #175

File: README.md [hunk 4]
```diff
@@ -6,111 +6,171 @@
...
-     │ String15        Float64  Float64      
-─────┼───────────────────────────────────────
-   1 │ Los Angeles     633.511  21.3912
-   2 │ Ventura         707.172  31.6856
-   3 │ Kern            678.235  53.1337
-  ⋮  │       ⋮            ⋮          ⋮
-  39 │ Mendocino       632.018   1.04942
-  40 │ Butte           627.0     0.0
-```
+## Future goals
```
Comment (ayushpatnaikgit) [inline README.md:164]:
```suggestion
## Goals
```

**Review by ayushpatnaikgit — APPROVED** [review]

**Comment by iuliadmtru** [comment]
Since this PR includes commits from the previously merged one (#175), I am closing it and creating a proper PR for the README update.

**Pull request closed without merging**